react/features/chat/components/native/Chat.js

```diff
@@ -33,15 +31,13 @@ class Chat extends AbstractChat<Props> {
     render() {
         return (
             <JitsiModal
-                headerLabelKey = 'chat.title'
+                headerProps = {{
+                    headerLabelKey: 'chat.title'
+                }}
                 modalId = { CHAT_VIEW_MODAL_ID }>
-                <KeyboardAvoidingView
-                    behavior = 'padding'
-                    style = { styles.chatContainer }>
-                    <MessageContainer messages = { this.props._messages } />
-                    <MessageRecipient />
-                    <ChatInputBar onSend = { this.props._onSendMessage } />
-                </KeyboardAvoidingView>
+                <MessageContainer messages = { this.props._messages } />
+                <MessageRecipient />
+                <ChatInputBar onSend = { this.props._onSendMessage } />
```

Comment:
shouldn't this be redered with the footerComponent prop and the Android-only padding hack restored!

Reply:
It doesn't need to be in the footer, unless we need to have an edge-to-edge backdrop, which is not the case here. And the padding hack is still there in the ChatInputBar